### PR TITLE
:octocat: @feature/contact-form

### PR DESCRIPTION
**Pull request by mahabubx7**
### PR: @feature/contact-form
**This branch `contact-feature` was developed by pair-programming.**

:heavy_check_mark: Added form elements
:heavy_check_mark: Added `formspree` to receive the emails.
:heavy_check_mark: Added input limitations for validate user inputs.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -238,10 +238,10 @@ <h4>skills</h4>
...
         <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-        <form action="#" id="contactForm">
-          <input type="text" name="full_name" placeholder="Full name">
-          <input type="email" name="email" placeholder="Email address">
-          <textarea name="message" id="contactMsg" cols="30" rows="10" placeholder="Enter text here"></textarea>
+        <form action="https://formspree.io/f/moqzdlze" id="contactForm" method="POST">
+          <input type="text" name="full_name" placeholder="Full name" required maxlength="30">
+          <input type="email" name="email" placeholder="Email address" required>
+          <textarea name="message" id="contactMsg" maxlength="500" cols="30" rows="10" placeholder="Enter text here" required></textarea>
           <button type="submit" class="btn btn-link">Get in touch</button>
```
Comment (topeogunleye) [inline index.html:245]: - [ ] I kindly suggest that you add the button interaction for  active (when it's pressed) as shown in the Figma template below:

image

the button on the pressed state has a slightly darker background: `#008552`
### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #008552;
}
```

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @mahabubx7, @Bestbynature, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mahabubx7) [reply index.html:245]: :heavy_check_mark: Got this :eyeglasses: 
Coming back in few minutes :+1:

**Comment by topeogunleye** [comment]
> Can you please re-check ❓ @topeogunleye

@mahabubx7 It has the active button interaction now 👍

**Comment by mahabubx7** [comment]
> 

So, it is going to be approved?

**Comment by topeogunleye** [comment]
> > 
> 
> So, it is going to be approved?

kindly submit for review. I guess it should

**Comment by mahabubx7** [comment]
> > > 
> > 
> > 
> > So, it is going to be approved?
> 
> kindly submit for review. I guess it should

:ok: Got it.

**Review by AdnanOlarmmi — APPROVED** [review]
Hi @team,

This is Adnan Is-haq, your beloved code reviewer, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved office man

## Highlights
- Nice working in pair.
- Professional README.
- Nice UI design.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mahabubx7** [comment]
Thank you @AdnanOlarmmi :heart_decoration:

**Pull request merged**